CHANGELOG.md

```diff
@@ -96,6 +105,14 @@ def f(t: tuple[int, int] | tuple[int, int, int]) -> None:
 
 This feature was contributed by Ivan Levkivskyi (PR [16237](https://github.com/python/mypy/pull/16237)).
 
+#### More Precise Tuple Lenghts (Experimental)
```

Comment:
```suggestion
#### More Precise Tuple Lengths (Experimental)
```

Reply:
Fixed in https://github.com/python/mypy/pull/16449.